ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -467,6 +479,9 @@ sub manage_users
 
     foreach my $user (@$users) {
         if ($user->{user}) {
+            if ($user->{user} eq "serveradmin" && exists $user->{password}) {
```

Comment:
This is a special case, serveradmin and oneadmin are internal system users used by OpenNebula, so if the component detects this username in your tree it changes the pass by the new one. I will include this info into the documentation to be more clear.